cljfmt/test/cljfmt/core_test.cljc

```diff
@@ -4,289 +4,750 @@
             [cljfmt.core :refer [reformat-string default-line-separator
                                  normalize-newlines find-line-separator
                                  replace-newlines wrap-normalize-newlines]]
-            [clojure.string :as str]))
+            [cljfmt.test-util.common :as common])
+  #?(:clj (:require [cljfmt.test-util.clojure])
+     :cljs (:require-macros [cljfmt.test-util.cljs])))
+
+;;; `reformats-to?` makes `reformat-string` tests more readable.
```

Comment:
I think these lines can be removed, as the tests themselves are good examples of use.